src/proposal.jl

```diff
@@ -94,12 +155,9 @@ function propose(rng::AbstractRNG, nt::NoUTurnTrajectory{I}, h::Hamiltonian, θ:
         n = n + n′
         s = s′ * (dot(θp - θm, ∂H∂r(h, rm)) >= 0 ? 1 : 0) * (dot(θp - θm, ∂H∂r(h, rp)) >= 0 ? 1 : 0)
         j = j + 1
-
-        # da_stats = α / nα
-
     end
 
-    return θ_new, r_new, 2^j
+    return θ_new, r_new, α / nα
```

Comment:
Currently only return accept ratio. I would make this in-sampling stats rich in another PR by introducing type for returned in-sampling stats.

Reply:
sounds good!